CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## vNext (TBD)
+## 10.4.0 (2021-08-30)
```

Comment:
Is this meant to be the date we started working in this version or will it be updated when it's released?

Reply:
The date when it was released - I started the workflow yesterday and forgot to wrap up the release. So I'll update.